WordPress/gutenberg
# Navigation Screen: Indicate when a menu is deleted and show the menu switcher afterwards

## Description

Closes https://github.com/WordPress/gutenberg/issues/25435

According to [comment](https://github.com/WordPress/gutenberg/issues/25435#issuecomment-776993797) the following steps have been added:
- When a menu is being deleted (call to the API is being made) a `delete` button has a `busy` status.
- All elements are dimmed when a call is being made
- A snack bar is shown, informing a user that a menu has been deleted
- When the API call is finished, a panel to choose another menu is displayed

## How has this been tested?



Tested manually.

I order to test:
1. Navigate to Gutenberg > Navigation (beta)
2. Select the menu area, and the navbar will appear
3. Click the `delete menu` button in the navbar
4. When a prompt asking, if a menu should be deleted, appears then click on the `OK` button


## Types of changes




New feature

## Checklist:
